packages/manager/src/components/VerticalLinearStepper/VerticalLinearStepper.styles.ts

```diff
@@ -0,0 +1,53 @@
+import { StepConnector, StepIcon } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+import { isPropValid } from 'src/utilities/isPropValid';
+
+type StyledCircleIconProps = {
+  activeStep: number;
+  index: number;
+};
+
+export const StyledCircleIcon = styled('div', {
+  label: 'StyledCircleIcon',
+  shouldForwardProp: (prop) => isPropValid(['backgroundColor', 'border'], prop),
+})<StyledCircleIconProps>(({ theme, ...props }) => ({
+  alignItems: 'center',
+  backgroundColor:
+    props.index === props.activeStep
+      ? theme.palette.primary.main
+      : props.index < props.activeStep
+      ? '#ADD8E6' // TODO: need UX confirmation on color code
+      : theme.bg.bgPaper, // Adjust colors as needed
+  border:
+    props.index < props.activeStep || props.index === props.activeStep
+      ? `2px solid ${theme.palette.primary.main}`
+      : `2px solid ${theme.borderColors.borderTable}`, // Adjust border styles as needed
+  borderRadius: '50%',
+  display: 'flex',
+  height: '24px',
```

Comment:
Maybe we can use theme.spacing() instead to avoid hardcoding values?

```suggestion
  height: theme.spacing(3),
```